Makefile

```diff
@@ -82,15 +68,70 @@ go_webdriver_test: go_webdriver_deps bower_components
 			--firefox_path=$(FIREFOX_PATH) \
 			--geckodriver_path=$(GECKODRIVER_PATH)
 
+sys_update: sys_deps
+	sudo apt-get update
+	gcloud components update
+	npm install -g npm
+
 go_webdriver_deps: go_deps webdriver_deps
 
 webdriver_deps:
+	sudo apt-get install --assume-yes --no-install-suggests default-jdk wget xvfb $$(apt-cache depends firefox-esr chromedriver |  grep Depends | sed "s/.*ends:\ //" | tr '\n' ' ')
+	cd $(WPTD_PATH)webapp; npm install web-component-tester --unsafe-perm
 	cd $(WEBDRIVER_PATH); ./install.sh $(BROWSERS_PATH)
 
-go_deps: $(find .  -type f | grep '\.go$' | grep -v '\.pb.go$')
+go_deps: sys_deps $(find .  -type f | grep '\.go$' | grep -v '\.pb.go$')
```

Comment:
Does `$(find ...)` work in `Makefile`? I think we need `$(shell find ...)`.

Besides, can we simply use `$(GO_FILES)` here? (`$(GO_FILES)` doesn't filter out `.pb.go`, but I don't think we have any of those, do we?).